docker/Dockerfile-workers

```diff
@@ -7,6 +7,7 @@ RUN dnf -y install \
     git \
     golang \
     make \
+    memcached \
```

Comment:
This looks to be in the wrong space in the Dockerfile since this is a multistage build. Try putting it in the second stage of the build.